.gitignore

```diff
@@ -10,11 +10,15 @@ node_modules/
 *.vk
 *.r1cs
 *.wtns
-*.key
 *.dat
 out
 witness
 proof.json
 abi.json
+proving.key
```

Comment:
Proposing to commit the various artifacts of running zokrates (which are themselves volatile). I can see an argument against and don't feel strongly about it.